tests/test_main.py

```diff
@@ -226,15 +226,7 @@ def test_predict_image_fromarray(m):
         prediction = m.predict_image(image=image)
 
     assert isinstance(prediction, pd.DataFrame)
-    assert set(prediction.columns) == {"xmin", "ymin", "xmax", "ymax", "label", "score"}
-
-
-def test_predict_return_plot(m):
-    image = get_data(path="2019_YELL_2_528000_4978000_image_crop2.png")
-    image = np.array(Image.open(image))
-    image = image.astype('float32')
-    plot = m.predict_image(image=image, return_plot=True)
-    assert isinstance(plot, np.ndarray)
+    assert set(prediction.columns) == {"xmin", "ymin", "xmax", "ymax", "label", "score","geometry"} 
```

Comment:
Space after comma (sorry; figure we're making changes might as well cleanup the little stuff)